lib/node_modules/@stdlib/math/base/complex/cis/test/fixtures/julia/runner.jl

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env julia
+
+import JSON
+
+"""
+    gen( re, im, name )
+
+Generate fixture data and write to file.
+
+# Arguments
+
+* `re`: real components for first complex number
+* `im`: imaginary components for first complex number
+* `name::AbstractString`: output filename
+
+# Examples
+
+``` julia
+julia> re = rand( 1000 );
+julia> im = rand( 1000 );
+julia> gen( re, im, \"data.json\" );
+```
+"""
+function gen( re, im, name )
+	cisre = Array( Float64, length(re) );
+	cisim = Array( Float64, length(re) );
+	for i in eachindex(re)
+		z = cis(Complex128( re[i], im[i] ));
+		cisre[ i ] = real( z );
+		cisim[ i ] = imag( z );
+	end
+
+	# Store data to be written to file as a collection:
+	data = Dict([
+		("re", re),
+		("im", im),
+		("cisre", cisre),
+		("cisim", cisim),
+	]);
+
+	# Based on the script directory, create an output filepath:
+	filepath = joinpath( dir, name );
+
+	# Write the data to the output filepath as JSON:
+	outfile = open( filepath, "w" );
+	write( outfile, JSON.json(data) );
+	close( outfile );
+end
+
+# Get the filename:
+file = @__FILE__;
+
+# Extract the directory in which this file resides:
+dir = dirname( file );
+
+# Pure nonnegative real
+re = rand( 500 ) * 10000.0 - 5.0;
```

Comment:
Parens: `( rand( 500 ) * 10000.0 ) - 5.0`.

Here and below. Follows the JavaScript convention.

Reply:
✅